From 1d6438de8a45b817c2e945b28af45dedc6b34020 Mon Sep 17 00:00:00 2001
From: Johannes Nau <johannes.nau@outlook.com>
Date: Fri, 26 May 2023 23:14:15 +0000
Subject: [PATCH] hotfix msid

---
 src/aiortc/sdp.py | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/src/aiortc/sdp.py b/src/aiortc/sdp.py
index d0fe057..70e1da6 100644
--- a/src/aiortc/sdp.py
+++ b/src/aiortc/sdp.py
@@ -561,11 +561,7 @@ class SessionDescription:
         assert media in self.media
         if media.msid is not None and " " in media.msid:
             bits = media.msid.split()
-            for group in self.msid_semantic:
-                if group.semantic == "WMS" and (
-                    bits[0] in group.items or "*" in group.items
-                ):
-                    return bits[1]
+            return bits[1]
         return None
 
     def __str__(self) -> str:
